Doc/library/enum.rst

```diff
@@ -573,6 +589,20 @@ Data Types
       better support the *replacement of existing constants* use-case.
       :meth:`__format__` was already :func:`int.__format__` for that same reason.
 
+.. class:: ReprEnum
+
+   :class:`!ReprEum` uses the :meth:`repr() <Enum.__repr__>` of :class:`Enum`,
+   but the :class:`str() <str>` of the mixed-in data type:
+
+      * :meth:`int.__str__` for :class:`IntEnum` and :class:`IntFlag`
+      * :meth:`str.__str__` for :class:`StrEnum`
```

Comment:
```suggestion
      * :meth:`!int.__str__` for :class:`IntEnum` and :class:`IntFlag`
      * :meth:`!str.__str__` for :class:`StrEnum`
```

Seems like this one might have been missed when my suggestions were apparently being re-implemented manually